shell/initialize/index.js

```diff
@@ -174,29 +170,22 @@ async function createApp(ssrContext, config = {}) {
   // Make app available into store via this.app
   store.app = app;
 
-  const next = ssrContext ? ssrContext.next : (location) => app.router.push(location);
+  const next = (location) => app.router.push(location);
   // Resolve route
-  let route;
-
-  if (ssrContext) {
-    route = router.resolve(ssrContext.url).route;
-  } else {
-    const path = getLocation(router.options.base, router.options.mode);
 
-    route = router.resolve(path).route;
-  }
+  const path = getLocation(router.options.base, router.options.mode);
+  const route = router.resolve(path).route;
 
   // Set context to app.context
   await setContext(app, {
     store,
     route,
     next,
     error:           app.nuxt.error.bind(app),
-    payload:         ssrContext ? ssrContext.payload : undefined,
-    req:             ssrContext ? ssrContext.req : undefined,
-    res:             ssrContext ? ssrContext.res : undefined,
-    beforeRenderFns: ssrContext ? ssrContext.beforeRenderFns : undefined,
-    ssrContext
+    payload:         undefined,
+    req:             undefined,
+    res:             undefined,
+    beforeRenderFns: undefined
```

Comment:
Can these be removed?

Reply:
I am unsure about `payload`, `req` and `res` but `beforeRenderFns` has no hits on the code so I assume it's safe to remove it.